frontend/src/app/shell.css

```
@@ -105,8 +114,13 @@
   padding-left: env(safe-area-inset-left);
   padding-right: env(safe-area-inset-right);
   /* Collapsing animates the grid column so the sidebar and the content edge
-     move together; labels cross-fade with it. */
-  transition: --railW var(--dur-move) var(--ease-out);
+     move together; labels cross-fade with it. The context column folds on the
+     same duration and the same curve, because both are the same gesture — a
+     column of the window opening or closing — and two speeds would read as two
+     unrelated mechanisms. */
+  transition:
+    --railW var(--dur-move) var(--ease-out),
```

Stylelint (17.14.0)

[error] Expected "--railW" to be "--railw" (value-keyword-case)

(value-keyword-case)